src/Components/Navtrans/Navtrans.jsx

```diff
@@ -175,19 +176,15 @@ const Navbar = styled.nav`
 
 const NavBottom = styled.nav`
   display: flex;
-  justify-content: center;
-  height: 35px;
-  margin: 10px 300px 0 0;
-
-  @media screen and (max-width: 1000px) {
-    margin: 10px 0 0 0;
-  }
+  max-width: 1060px;
+  height: 50px;
+  margin: 0 auto;
 
   .categoryItem {
     padding: 0 10px;
 
     p {
-      padding: 0 7px 15px 7px;
+      padding: 15px 7px;
       color: ${(props) => (props.themeColor === "normal" ? theme.darkGray : "rgba(255,255,255,0.7)")};
```

Comment:
```suggestion
      color: ${({ themeColor, theme }) => (themeColor === "normal" ? theme.darkGray : "rgba(255,255,255,0.7)")};
```
- themeProvider 사용하여 theme 객체를 받을 수 있습니다.
- 함수 파라미터 자리에서 직접 구조분해 할당 해줄 수도 있습니다.

Reply:
호곡...여기서도 구조분해할당이...신세계...수정하겠슴다!